<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Molecular Biology Guide for Fucking Idiots</title>
    <link rel="stylesheet" type="text/css" href="tech.css" />
  </head>

  <body>
    <nav id="navbar">
      <header>Restriction Enzyme Cloning Manual</header>
      <ul>
        <li>
          <a
            href="#Selecting_the_Cloning_System_and_Plasmid_Vector"
            class="nav-link"
            id="nav-link"
            >Selecting the Cloning System and Plasmid Vector</a
          >
        </li>
        <li>
          <a
            href="#Plasmid_Restriction_Digestion"
            class="nav-link"
            id="nav-link"
            >Plasmid Restriction Digestion</a
          >
        </li>
        <li>
          <a
            href="#Gel_Excission_of_DNA_Fragments"
            class="nav-link"
            id="nav-link"
            >Gel Excission of DNA Fragments</a
          >
        </li>
        <li>
          <a href="#Clean_Up_of_Gel_Fragments" class="nav-link" id="nav-link"
            >Clean Up of Gel Fragments</a
          >
        </li>
        <li>
          <a
            href="#Site_Directed_Mutagenesis_Protocol"
            class="nav-link"
            id="nav-link"
            >Site Directed Mutagenesis Protocol</a
          >
        </li>
      </ul>
    </nav>

    <main id="main-doc">
      <div class="card">
        <section
          id="Selecting_the_Cloning_System_and_Plasmid_Vector"
        >
          <header>Selecting the Cloning System and Plasmid Vector</header>
          <article>
            <p>
              Genetic engineering is used in thousands of laboratories around
              the world. Given its importance it is remarkable that cloning
              strategies for many of the popular DNA components are not
              standardised. Contributing to this lack of standardization are the
              many options available for cloning kits and innovative
              technologies. It is not always necessary to use an IP- heavy kit,
              as traditional cloning by restriction enzyme digestion is made
              even easier with standardized plasmids from Oxford Genetics.
            </p>

            <p>
              The aim at Oxford Genetics was to engineer a DNA plasmid system
              that could accommodate most of the functional DNA inserts that a
              researcher might require within a single plasmid. By optimising
              our starting vector, every DNA component of our system can be
              removed and exchanged for hundreds of other DNA sections that have
              been pre-designed and tested by us. This is the SnapFast™ concept.
              All of our constructs have been pre-screened for poor codon usage
              and conflicting restriction sites. Where possible, rare codons and
              restriction sites have been removed to enable efficient
              expression, and ensure that restriction sites do not limit the
              cloning of other SnapFast DNA sections.
            </p>
          </article>
        </section>
      </div>

      <div class="card">
        <section class="main-section" id="Plasmid_Restriction_Digestion">
          <header>Plasmid Restriction Digestion</header>
          <article>
            <p>
              A preparative digest is the cutting of DNA to prepare it for
              ligation with another piece of DNA, not simply to confirm the
              identity of the DNA. You should aim to set up the digest of both
              your fragment and vector at the same time. This will save you
              time. This protocol assumes a plasmid solution in Tris-EDTA (TE)
              or Elution buffer (EB) or nuclease free H2O that is 200-300 ng/µl
              and about 3-5 kb. Larger plasmids may require you to use more
              volume because at the same concentration there will be less
              plasmid copies.
            </p>

            <p>
              In this particular example, the following could be done:
            </p>

            <ol>
              <li>
                10-20 µl of plasmid
                <code>(200-300 ng/μl, total amount 3-5 µg)</code>
              </li>
              <li>
                10 µl of 10X Restriction buffe
              </li>
              <li>
                10 µl of 10X Bovine Serum Albumin (BSA, final concentration is
                usually 100 µg/ml)
                <code>(Catalog No. A7906)</code>
              </li>
              <li>
                1.5-2 µl Restriction Enzyme 1
                <code>(10-20 units/µl)</code>
              </li>
              <li>
                Make up to 70 or 100µl total volume with TE (Catalog No. T9285)
                or nuclease free water
                <code>(Catalog No. W4502)</code>
              </li>
            </ol>

            <p>
              Add the reagents above in a sterile 1.5ml Eppendorf, first add the
              TE or water, then the plasmid/DNA, then the restriction buffer and
              BSA, and mix thoroughly. Finally, add the restriction enzyme(s).
              The restriction enzymes chosen depend on your goals and the
              plasmid map, but may include EcoR I, BamH I, or Hind III. Incubate
              the reaction at the right temperature (usually 37 °C, but always
              make sure that you are not dealing with an exception, like SwaI at
              25 °C) and start your alarm clock (count up). Run the reaction for
              40-60 minutes. Now you can de-phosphorylate the vector using
              alkaline phosphatase (<code>CIP</code>). You may wish to
              deactivate the restriction enzymes at his point, this is standard
              practice.
            </p>
          </article>
        </section>
      </div>

      <div class="card">
        <section class="main-section" id="Gel_Excission_of_DNA_Fragments">
          <header>Gel Excission of DNA Fragments</header>
          <article>
            <p>
              This is a protocol for a preparative digest, which is the cutting
              of DNA to prepare it for ligation with another piece of DNA, not
              simply to confirm the identity of the DNA. Usually these pieces of
              DNA are circular plasmids but they could also be PCR fragments.
              You should aim to set up the digest of both your fragment and
              vector at the same time. This will save you time later.
            </p>

            <p>
              The standard method used to extract DNA from agarose gels is by
              visualization of ethidium bromide (Catalog No. H5041) stain with
              UV light. This is not ideal because the UV will damage the DNA and
              can cause mutations and can reduce the ligation efficiency. An
              alternative is to use a Clare Chemical Dark Reader
              Trans-illuminator but it can sometimes be difficult to see low
              quantity bands. If you have a signficant amount of DNA these
              systems are ideal. Alternatively, you can do a blind excision by
              running a small quantity of your cut DNA in the lane adjacent to
              your large well, and use its position to excise a large quantity
              of DNA in the next well. Be very careful with this technique as
              some bands can be quite small lengthways/top-to-bottom in the gel
              and can be missed if you are not accurate. After excision you can
              look at the remaining gel to see if you got the band, do it
              quickly because if you didn't get the DNA you don’t want to damage
              it with the UV.
            </p>

            <p>
              Be careful of DNA fragments that are close together. The key of
              this technique is to separate the vector backbone from the
              fragment to prevent contamination. Just because you cannot see
              something when you are excising doesn’t mean it isn’t there, you
              will only see the peak of a Gaussian distribution of DNA and
              whilst the bands may look separate they may not be and you may get
              some DNA from bands that are close by accident.
            </p>
          </article>
        </section>
      </div>

      <div class="card">
        <section class="main-section" id="Clean_Up_of_Gel_Fragments">
          <header>Clean Up of Gel Fragments</header>
          <article>
            <p>
              DNA gel extraction kits usually consist of either beads or spin
              columns. The sizes of DNA fragments that each kit can accommodate
              can vary, so check that the kit will extract the size range of DNA
              you are working with. Most kits cover from 100bp – 5 Kb but some
              have lower or higher capacities.
            </p>

            <p>
              The principle of both spin column and bead kits is to bind the DNA
              to something (usually a silica resin) and then wash away the
              contaminants, either by pelleting the beads, or washing the
              column. The final elution step releases the DNA from the
              resin/beads/column to provide you with a clean DNA fragment.
              However, even the best extraction kit can still carry though some
              contaminants and, for this reason, using large volumes of a
              solution that has been produced by gel extraction in a ligation
              should be avoided. Sometimes ‘adding less is more’ when it comes
              to adding your fragment to your ligation. Try to avoid making a
              ligation consist of more than 30% volume of liquid isolated by gel
              extraction.
            </p>
          </article>
        </section>
      </div>

      <div class="card">
        <section class="main-section" id="Site_Directed_Mutagenesis_Protocol">
          <header>Site Directed Mutagenesis Protocol</header>
          <article>
            <p>
              Site Directed Mutagenesis (SDM) is a useful technique for
              introducing a specific mutation into a plasmid at a specific site.
              The mutation can be a substitution, insertion or deletion. There
              are many applications for SDM, for instance to assess the function
              of certain amino acids in an enzyme, to investigate the effect of
              altering bases in a promoter or removing restriction sites from a
              plasmid.
            </p>

            <p>
              Here we describe a PCR based site directed mutagenesis method. The
              basic principle is to design a pair of PCR primers back to back,
              so that the entire plasmid is amplified by PCR. One of these
              primers incorporates the desired mutation. The PCR creates a
              linear product whose ends can then be joined together (after
              phosphorylation) with DNA ligase. The circularised vector is
              transformed into E. coli.
            </p>

            <p>
              PCR primers normally come without a phosphate group on their 5'
              termini due to the way they are synthesised. This means that the
              ends of a PCR product cannot simply be ligated together, they must
              be phosphorylated first. There are two main options for this: 1)
              you could order the primers with a phosphate already added to the
              5' end, or 2) you could phosphorylate the PCR product using
              polynucleotide kinase (PNK). Phosphorylated primers are a good
              idea if you are only doing a few SDMs and don't have any PNK in
              the freezer (plus it cuts out a step). Using PNK is more cost
              effective if you're doing a lot of SDMs (probably 10 or more).
            </p>
          </article>
        </section>
      </div>
      <div class="footer">
        <p>
          &#169; 2020 Simon &quot;The Molecular Spectre of Genetic
          Engineering&quot; Trueteller <br />
          <span>All rights reserved</span>
        </p>
      </div>
    </main>
  </body>
</html>
